client/components/CandidateReview/CandidateTestPlanRun/InstructionsRenderer.jsx

```diff
@@ -169,19 +170,152 @@ const InstructionsRenderer = ({
         commandsContent
     );
 
-    const assertions = [...pageContent.instructions.assertions.assertions];
-    const assertionsContent = parseListContent(assertions);
-
     const Heading = `h${headingLevel}`;
 
     return (
         <>
             <NumberedList>{allInstructionsContent}</NumberedList>
-            <Heading>{pageContent.instructions.assertions.header}</Heading>
-            {pageContent.instructions.assertions.description}
-            <NumberedList>{assertionsContent}</NumberedList>
             {settingsContent.length ? settingsContent : null}
 
+            {renderableContent.commands.map(
+                ({ id, settings, assertionExceptions = [] }, i) => {
+                    const settingsScreenText = isV2
+                        ? renderableContent.target.at.raw.settings[settings]
+                              ?.screenText ?? ''
+                        : null;
+
+                    let mustCount = 0;
+                    let shouldCount = 0;
+                    let mayCount = 0;
+
+                    let assertions = [...renderableContent.assertions];
+
+                    assertionExceptions.forEach(exception => {
+                        const assertionIndex = assertions.findIndex(
+                            assertion => {
+                                return (
+                                    assertion.assertionId ===
+                                    exception.assertionId
+                                );
+                            }
+                        );
+                        if (exception.priority === 0) {
+                            assertions.splice(assertionIndex, 1);
+                        } else {
+                            assertions[assertionIndex].priority =
+                                exception.priority;
+                        }
+                    });
+
+                    assertions.forEach(({ priority }) => {
+                        const priorityString =
+                            convertAssertionPriority(priority);
+                        if (priorityString === 'MUST') mustCount += 1;
+                        if (priorityString === 'SHOULD') shouldCount += 1;
+                        if (priorityString === 'MAY') mayCount += 1;
+                    });
+
+                    let settingsScreenTextFormatted;
+
+                    if (
+                        settingsScreenText === null ||
+                        settingsScreenText === ''
+                    ) {
```

Comment:
Both of these values are falsy so this could be `if (!settingsScreenText) {` or even more compact:

```js
const settingsScreenTextFormatted = settingsScreenText ? ` (${settingsScreenText})` : ''
```